contrib/ansible/deploy-playbook.yml

```diff
@@ -38,13 +38,20 @@
       cluster_operator_namespace: "{{ cli_cluster_operator_namespace }}"
     when: cli_cluster_operator_namespace is defined
 
+  - name: check if cluster operator namespace exists
```

Comment:
It's unfortunate that we have to go through these machinations. Isn't the main point of Ansible that it will handle the exists/not-exists for you?

Reply:
The complication here is that we are leaning on kube to be idempotent with the request by using kubectl apply (which we should be doing), but we do not have permissions to create namespaces, so that apply fails regardless.